python/cudf/cudf/utils/_numba.py

```diff
@@ -0,0 +1,196 @@
+# Copyright (c) 2023, NVIDIA CORPORATION.
+
+import glob
+import math
+import os
+import subprocess
+import sys
+import warnings
+
+from numba import config
+
+CC_60_PTX_FILE = os.path.dirname(__file__) + "/../core/udf/shim_60.ptx"
+_NO_DRIVER = (math.inf, math.inf)
+
+CMD = """\
```

Comment:
Use a descriptive name.
```suggestion
CHECK_CUDA_VERSION_CMD = """\
```